projects/golang/go/1.17/rpmbuild/SPECS/golang.spec

```diff
@@ -164,6 +164,12 @@ Patch8:   0008-go-1.17.13-eks-multipart-limit-memory-inode-consumption.patch
 Patch9:   0009-go-1.17.13-eks-path-filepath-do-not-clean-relative-path.patch
 Patch10:  0010-go-1.17.13-eks-update-bundled-golang-org.patch
 Patch11:  0011-go-1.17.13-eks-net-http-update-bundled-golang.patch
+Patch12:  0012-go-1.17.13-eks-go-scanner-reject-large-line-a.patch
+Patch13:  0013-go-1.17.13-mime-multipart-avoid-excessive.patch
+Patch14:  0014-go-1.17.13-eks-net-textproto-mime-multipart-i.patch
+Patch15:  0015-go-1.17.13-eks-mime-multipart-limit-parsed-mi.patch
+Patch105:  0105-go-1.17.13-eks-add-Cut-method-to-bytes-package.patch
```

Comment:
Patches are applied in this order. Hence, Utility patch is placed before the CVE fix patch.